### PR TITLE
Feature/comprison tool

### PR DESCRIPTION
**Pull request by jianmingtu**
# Description

This PR includes the following proposed change(s):

- {test getCriminalFile in comparison tool}

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: comparison-tool-1.1/src/main/java/ca/bc/gov/open/ccd/comparison11/services/TestService.java [hunk 1]
```diff
@@ -0,0 +1,372 @@
...
+        fileOutput =
+                new PrintWriter(
+                        outputDir + " getCriminalFileContentRoomCode.txt", StandardCharsets.UTF_8);
+
+        while (scanner.hasNextLine()) {
+            String[] line = scanner.nextLine().split(",");
+            String roomCode = "", procDate = "", identCd = "";
+            for (int i = 0; i < line.length; i++) {
+                if (line[i] != null) {
+                    if (RoomCodeCompare.COURT_ROOM_CODE.ordinal() == i) roomCode = line[i];
```
Comment (ekwanNTT) [inline comparison-tool-1.1/src/main/java/ca/bc/gov/open/ccd/comparison11/services/TestService.java:148]: This need brackets!

File: comparison-tool-1.1/src/main/resources/application.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+host:
+  password: ${BASIC_AUTH_PASS}
+  username: ${BASIC_AUTH_USER}
+  api-host: ${API_HOST}
+  wm-host: ${WM_HOST}
+server:
```
Comment (ekwanNTT) [inline comparison-tool-1.1/src/main/resources/application.yml:6]: remove this

File: comparison-tool-1.1/src/main/java/ca/bc/gov/open/ccd/comparison11/services/TestService.java (hunk 1 above)
Comment (jianmingtu) [reply comparison-tool-1.1/src/main/java/ca/bc/gov/open/ccd/comparison11/services/TestService.java]: done

File: comparison-tool-1.1/src/main/resources/application.yml (hunk 2 above)
Comment (jianmingtu) [reply comparison-tool-1.1/src/main/resources/application.yml:6]: removed

**Review by ekwanNTT — APPROVED** [review]

**Review by ekwanNTT — APPROVED** [review]

**Pull request merged**